alacritty/src/renderer/mod.rs

```diff
@@ -21,24 +22,28 @@ use std::ptr;
 use std::sync::mpsc;
 use std::time::Duration;
 
-use fnv::FnvHasher;
 use font::{
     self, BitmapBuffer, FontDesc, FontKey, GlyphKey, Rasterize, RasterizedGlyph, Rasterizer,
 };
+
+use fnv::FnvHasher;
 use log::{error, info};
 use notify::{watcher, DebouncedEvent, RecursiveMode, Watcher};
```

Comment:
`use font` should be below foreign crates and before `use crate`.

The `alacritty_terminal` should also be above `crate`, but below foreign crates or our own crates.